### PR TITLE
Rhel pxk fips

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR provides guidance on how to create a custom image containing RHEL for vSphere. The image can be used with the BYOOS pack along with the PXK distro. 

- [RHEL & PXK](https://deploy-preview-2728--docs-spectrocloud.netlify.app/byoos/usecases/vmware/rhel-pxk/)

Other changes include a reference table for BYOOS usecases and the value to assign to the `osImageOverride` field.
- [BYOOS](https://deploy-preview-2728--docs-spectrocloud.netlify.app/integrations/byoos/?edge-non-edge=Non-Edge#reference-custom-image)

## Changed Pages

💻 [Preview URL for Page](https://deploy-preview-2728--docs-spectrocloud.netlify.app)

## Jira Tickets

🎫 [DOC-1078](https://spectrocloud.atlassian.net/browse/DOC-1078)
🎫 [DOC-1136](https://spectrocloud.atlassian.net/browse/DOC-1136)
🎫 [DOC-1162](https://spectrocloud.atlassian.net/browse/DOC-1162)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-1078]: https://spectrocloud.atlassian.net/browse/DOC-1078?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOC-1136]: https://spectrocloud.atlassian.net/browse/DOC-1136?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOC-1162]: https://spectrocloud.atlassian.net/browse/DOC-1162?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by addetz — COMMENTED** [review]
Some minor comments that I found through testing on VMware. Will test again on Monday on a larger instance and send the approval through once I complete that 🚀

File: docs/docs-content/byoos/usecases/vmware/rhel-pxk.md [hunk 1]
```diff
@@ -0,0 +1,559 @@
...
+      osVersion: "8.8"
+    ```
+
+    Using the following image, take note of the three numbers highlighted in the image name.
+
+    1. The BYOOS pack is selected as the operating system layer.
+    2. The custom RHEL image template is specified in the `osImageOverride` field.
+    3. The Kubernetes version is set to `1.27.11`, the same version specified in the `kubernetes.json` file.
+
+    ![A cluster profile using a custom RHEL image](/byoos_vmware_rhewl-pxk_cluster-profile-non-fips.webp)
```
Comment (addetz) [inline docs/docs-content/byoos/usecases/vmware/rhel-pxk.md:515]: Your name is not blurred in this image.

File: docs/docs-content/byoos/usecases/vmware/rhel-pxk.md [hunk 2]
```diff
@@ -0,0 +1,559 @@
...
+- Access to the VMware vSphere environment, including user credentials allowing you to create virtual machines.
+
+- A valid [RHEL subscription](https://www.redhat.com/en/store/linux-platforms). You will need to provide the username
+  and password for the subscription during the build process.
+
+- An x86_64 Linux VM with the following resources:
+
+  - 4 CPU
+  - 8 GB of RAM
+  - 50 GB of free disk space
```
[the remark this answers is not in the document]
Comment (addetz) [reply docs/docs-content/byoos/usecases/vmware/rhel-pxk.md:33]: The final usage was 50.42GB when I retried on a larger instance.

**Review by addetz — APPROVED** [review]

**Comment by addetz** [comment]
Verified FIPS workflow with VerteX 4.3.5 🚀

File: docs/docs-content/byoos/usecases/vmware/rhel-pxk.md [hunk 3]
```diff
@@ -0,0 +1,559 @@
...
+
+## Build Custom Image
+
+Use the following steps to build a custom RHEL image with PXK for VMware vSphere. Select the tab based on whether you
+want to create a FIPS-compliant image or a non-FIPS-compliant image.
+
+<Tabs>
+<TabItem value="fips" label="FIPS">
+
+1.  Open a teminal session and log in to the Linux VM.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/byoos/usecases/vmware/rhel-pxk.md:50]:
```suggestion
1.  Open a terminal session and log in to the Linux VM.
```

File: docs/docs-content/byoos/usecases/vmware/rhel-pxk.md [hunk 4]
```diff
@@ -0,0 +1,559 @@
...
+    ![A cluster profile using a custom RHEL image](/byoos_vmware_rhewl-pxk_cluster-profile.webp)
+
+    Complete the remaining steps in the
+    [cluster profile creation process](../../../profiles/cluster-profiles/create-cluster-profiles/create-infrastructure-profile.md).
+    Use the cluster profile to deploy a Kubernetes cluster in your VMware vSphere environment with the custom RHEL
+    image.
+
+</TabItem>
+<TabItem value="non-fips" label="Non-FIPS">
+1. Open a teminal session and log in to the Linux VM.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/byoos/usecases/vmware/rhel-pxk.md:295]:
```suggestion
1. Open a terminal session and log in to the Linux VM.
```

**Pull request merged**